### PR TITLE
[WIP] Update changelogs

### PR DESCRIPTION
**Pull request by avilene**
For issue #1059

### CODE REVIEW

**Comment by kfinch** [comment]
Build errors need to be resolved, looks like you mistakenly removed the import for Bicepspump:
```
./src/Parser/DeathKnight/Unholy/CHANGELOG.js
  Line 14:  'Bicepspump' is not defined  no-undef
  Line 19:  'Bicepspump' is not defined  no-undef
```

**Comment by kfinch** [comment]
Still more import issues, according to Travis. Please remember to run the build locally and make sure it completes without errors before pushing a PR.

File: src/Parser/Mage/Frost/CHANGELOG.js [hunk 1]
```diff
@@ -4,91 +4,93 @@ import { Sharrq, sref } from 'MAINTAINERS';
...
-      contributors: [Sharrq],
-  },
+    {
+        date: new Date('2018-01-01'),
+        changes: <Wrapper>Added a 'Fish for procs' entry to the Checklist, and reworded/updated several suggestion metrics.</Wrapper>,
+        contributors: [sref],
+    },
+    {
+        date: new Date('2017-12-29'),
+        changes: <Wrapper>Added support for <SpellLink id={SPELLS.FROST_MAGE_T21_4SET_BONUS_BUFF.id} icon /></Wrapper>,
```
Comment (kfinch) [inline src/Parser/Mage/Frost/CHANGELOG.js]: Did you mistakenly add 'icon' at the end here? Not sure what 'icon' means.

**Review by kfinch — CHANGES_REQUESTED** [review]
Looks good, just found one typo. Also looks like there is a merge conflict in Fire Mage.

At this point I think you should focus on resolving the issues so we can get this merged, rather than adding more specs change logs. This is because by its nature the changelog changes a lot as people merge PRs, so you'll just continue to be hit by merge conflicts. You can always open another PR for the remaining specs to be converted.

File: src/Parser/Mage/Frost/CHANGELOG.js (hunk 1 above)
Comment (Pewtro) [reply src/Parser/Mage/Frost/CHANGELOG.js]: It shows the icon of the spell

Comment (kfinch) [reply src/Parser/Mage/Frost/CHANGELOG.js]: Oh my bad, yeah I misread it as being outside the SpellLink for some reason, and I also didn't know about the Icon part.

**Comment by yajinni** [comment]
HI! I updated my change log today. So you can ignore mine :P

**Pull request closed without merging**